certbot/cli.py

```diff
@@ -944,19 +953,17 @@ def _paths_parser(helpful):
         help="Logs directory.")
     add("paths", "--server", default=flag_default("server"),
         help=config_help("server"))
-    # overwrites server, handled in HelpfulArgumentParser.parse_args()
-    add("testing", "--test-cert", "--staging", action='store_true', dest='staging',
-        help='Use the staging server to obtain test (invalid) certs; equivalent'
-             ' to --server ' + constants.STAGING_URI)
 
 
 def _plugins_parsing(helpful, plugins):
+    # It's nuts, but there are two "plugins" topics.  Somehow this works
```

Comment:
To clarify, I don't think this should be done in this PR. It's just a bit awkward that `-h plugins` gives you help for two different things. If you agree, please open an issue or comment here and I'll do so.